programs/server/TelemetryCollector.h

```diff
@@ -19,7 +19,7 @@ class TelemetryCollector
     bool new_session = true;
     Int64 started_on_in_minutes;
 
-    static constexpr auto INTERVAL_MS = 60 * 60 * 1000; /// 1 hour
```

Comment:
Let's make it configurable, otherwise we will keep updating this interval since someday we may find 1) too much volume, 2) don't worth this frequency 3) users complain the frequency

Reply:
Let me merge this first and @qijun-niu-timeplus , please move this to config file in the other PR